.github/workflows/benchmarks.yml

```diff
@@ -34,76 +30,47 @@ jobs:
           cd packages/benchmarks
           npx tachometer \
             --config lit-html/kitchen-sink/tachometer.json \
-            --json-file lit-html/kitchen-sink/results.json
-
-      - name: Report lit-html-kitchen-sink
-        uses: andrewiggins/tachometer-reporter-action@v2
-        with:
-          report-id: lit-html-kitchen-sink
-          path: packages/benchmarks/lit-html/kitchen-sink/results.json
-          pr-bench-name: this-change
-          base-bench-name: tip-of-tree
+            --json-file lit-html-kitchen-sink.json
 
       # lit-html:template-heavy
       - name: Benchmark lit-html/template-heavy
         run: |
           cd packages/benchmarks
           npx tachometer \
             --config lit-html/template-heavy/tachometer.json \
-            --json-file lit-html/template-heavy/results.json
-
-      - name: Report lit-html-template-heavy
-        uses: andrewiggins/tachometer-reporter-action@v2
-        with:
-          report-id: lit-html-template-heavy
-          path: packages/benchmarks/lit-html/template-heavy/results.json
-          pr-bench-name: this-change
-          base-bench-name: tip-of-tree
+            --json-file lit-html-template-heavy.json
 
       # lit-html:repeat
       - name: Benchmark lit-html/repeat
         run: |
           cd packages/benchmarks
           npx tachometer \
             --config lit-html/repeat/tachometer.json \
-            --json-file lit-html/repeat/results.json
-
-      - name: Report lit-html-repeat
-        uses: andrewiggins/tachometer-reporter-action@v2
-        with:
-          report-id: lit-html-repeat
-          path: packages/benchmarks/lit-html/repeat/results.json
-          pr-bench-name: this-change
-          base-bench-name: tip-of-tree
+            --json-file lit-html-repeat.json
 
       # lit-element:list
       - name: Benchmark lit-element/list
         run: |
           cd packages/benchmarks
           npx tachometer \
             --config lit-element/list/tachometer.json \
-            --json-file lit-element/list/results.json
-
-      - name: Report lit-element-list
-        uses: andrewiggins/tachometer-reporter-action@v2
-        with:
-          report-id: lit-element-list
-          path: packages/benchmarks/lit-element/list/results.json
-          pr-bench-name: this-change
-          base-bench-name: tip-of-tree
+            --json-file lit-element-list.json
 
       # reactive-element:list
       - name: Benchmark reactive-element/list
         run: |
           cd packages/benchmarks
           npx tachometer \
             --config reactive-element/list/tachometer.json \
-            --json-file reactive-element/list/results.json
+            --json-file reactive-element-list.json
 
-      - name: Report reactive-element-list
-        uses: andrewiggins/tachometer-reporter-action@v2
+      - name: Upload results
```

Comment:
Nice! And this is what will be downloaded by the other action & reported as a comment.